### PR TITLE
Nicholas truson

### PR DESCRIPTION
**Pull request by NicholasTruson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,14 +30,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+The Document Object Model is an object representation of the html elements of a webpage.
+
```
Comment (rleslie1015) [inline README.md]: yes perfect!

File: README.md [hunk 2]
```diff
@@ -30,14 +30,24 @@ Edit this document to include your answers after each question. Make sure to lea
...
+A method where we can chose specific elements to listen to specific events and fire a callback when that event occurs.
+
 4. Why would we convert a NodeList into an Array?
 
+To be able to use Query Selector All and use Array Methods (foreach, etc.)
+
 5. What is a component?
 
+Components are reusable pieces of code that can be used to build elements sharing functionality and styling.
+
```
Comment (rleslie1015) [inline README.md]: All perfect answers

File: components/Cards/index.js [hunk 3]
```diff
@@ -17,3 +17,90 @@
...
+    Cards.appendChild(createCard(card.headline, card.authorPhoto, card.authorName));
+    })
+    })
+
+// *** Neg *** //
+
+.catch( error => {
+    console.log("This ain't working bro", error);
+    })
+
```
Comment (rleslie1015) [inline components/Cards/index.js]: YES! Great job on the axios Get request to retrieve data, followed by the looping of individual arrays to create articles. Perfect. If you  want to make your code dry you can loop through an object using a `for...in loop`. like this 
```
axios
  .get('https://lambda-times-backend.herokuapp.com/articles')
  .then(response => {
    let articles = response.data.articles;
    for (topic in articles) {
      articles[topic].forEach(article => {
        entry.appendChild(ArticleCard(article));
      });
    }
  });
```

**Review by thereactgirl — COMMENTED** [review]
Great job completing the objectives in just 3 hours!! YOU KILLED IT. Congrats 2 stars

**Pull request merged**